usage/test-framework/src/test/java/org/apache/brooklyn/test/framework/TestFrameworkAssertionsTest.java

```diff
@@ -19,6 +19,7 @@
 package org.apache.brooklyn.test.framework;
 
 import com.google.common.base.Supplier;
+import org.apache.brooklyn.test.Asserts;
```

Comment:
Not in the diff, but `import static org.assertj.core.api.Assertions.assertThat;` is unused.